src/services/index.js

```diff
@@ -1 +1,16 @@
-// déclarez ici les appels aux API de services métier
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadUser = async ({id}) => {
```

Comment:
For the loadUser you have to create a webService specific, like this, the "then" is not necessary